.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  architect: giantswarm/architect@0.8.4
+  architect: giantswarm/architect@0.9.0
```

Comment:
Only after architect-orb 0.8.15, we are upgrading go version to 1.14. So better to update this at the latest.